src/common/hooks/useFieldErrorMsg.tsx

```diff
@@ -23,7 +23,10 @@ const useFieldErrorMsg = ({ name, inputError, validate }: Props): string | undef
     }
   }, [field.value, setHadValue]);
   const showError = field.value || (!field.value && hadValue);
-  const errorMessage = (showError ? error : '') || inputError;
+  let errorMessage = (showError ? error : '') || inputError;
+  if (errorMessage) {
```

Comment:
Since this is a workaround to some unexpected data we are receiving, I think it's best to indicate why we are doing this change. Either define a function whose name implies it, or add a comment for it.

Otherwise someone may be tempted to remove this code since theoretically at this point, `errorMessage` must be a string already. 
Also, it may be better to split the lines using "dot" as the separator instead of  "comma" so they read as independent phrases and are easier to understand.